src/include/catalog/table_schema.h

```diff
@@ -31,6 +31,8 @@ class TableSchema {
 
     virtual bool isParent(common::table_id_t tableID) = 0;
 
+    virtual std::string toCypher(std::string srcTableName, std::string dstTableName) const = 0;
```

Comment:
toCypher taking srcTableName and dstTableName doesn't make sense to me. You can pass a `Catalog` here instead if you need to access tableName from tableID.